server/controllers/placesController.js

```diff
@@ -45,32 +40,27 @@ const createPlace = async (req, res) => {
 
 const updatePlace = async (req, res) => {
   const { id } = req.params;
-  if (!id) {
-    return res.json({
-      message: "Lugar no encontrado",
-    });
-  }
   try {
-    await place.update(req.body, {
+    const placeUpdated = await place.update(req.body, {
       where: {
```

Comment:
Debes asignarle al id el valor del id del request:
```suggestion
      where: {
      id: id
      }
```